lib/abide_dev_utils/errors/xccdf.rb

```diff
@@ -20,5 +20,9 @@ class ControlPartsError < GenericError
     class ProfilePartsError < GenericError
       @default = 'Failed to extract parts from profile name:'
     end
+
+    class ProfileUnsupportedError < GenericError
```

Comment:
This error is a bit ambiguous because XCCDF files don't actually have types, that's just a convention we use to refer to the compliance framework being represented in the XCCDF. This error also has nothing to do with CIS benchmark profiles, but it's named like it does.